lib/middleware/permissions.js

```diff
@@ -6,7 +6,7 @@ module.exports = (task, params) => (req, res, next) => {
         return next();
       }
       const err = new Error('Unauthorised');
```

Comment:
Should this be "Forbidden" then?

Reply:
If you're being _particularly_ anal then... maybe.

403 errors get handled with a bespoke error template anyway, so it doesn't matter. It depends how much of a stickler for the HTTP spec you are.

Reply:
well if we're going to be anal about using the correct error codes then we might as well send the accompanying message as well :)

Reply:
I'm more interested in "consistent" than "correct", but whatevs.